cmd/beaker/workspace.go

```diff
@@ -36,7 +36,8 @@ func newWorkspaceArchiveCommand() *cobra.Command {
 		Short: "Archive a workspace, making it read-only",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := beaker.Workspace(args[0]).SetArchived(ctx, true); err != nil {
+			trueVal := true
+			if _, err := beaker.Workspace(args[0]).Patch(ctx, api.WorkspacePatch{Archive: &trueVal}); err != nil {
```

Comment:
Nit: long lines don't format well in GitHub.
```suggestion
			if _, err := beaker.Workspace(args[0]).Patch(ctx, api.WorkspacePatch{
				Archive: &trueVal,
			}); err != nil {
```